NEWS.rst

```diff
@@ -1,6 +1,12 @@
 History
 =======
 
+0.7 (2018-09-20)
+----------------
+
+* Python 3 support
+* Clean up tests and more flake8 (line lengths)
```

Comment:
Add note about fixes for newer numpy?

Reply:
Since it only affects one test, it is not that special.